src/apps/competitions/models.py

```diff
@@ -322,6 +326,12 @@ def cancel(self):
             return True
         return False
 
+    def check_children(self):
```

Comment:
I'm not a huge fan of this name, something like `check_children_completed` is _maybe_ better.. hmm..